packages/webapp/pages/settings/organization/[orgId]/general.tsx

```diff
@@ -31,21 +31,62 @@ import type {
   UpdateOrganizationInput,
 } from '@dailydotdev/shared/src/features/organizations/types';
 import { SubscriptionStatus } from '@dailydotdev/shared/src/lib/plus';
-import { AccountPageContainer } from '../../../../components/layouts/SettingsLayout/AccountPageContainer';
-import { defaultSeo } from '../../../../next-seo';
-import { getTemplatedTitle } from '../../../../components/layouts/utils';
+import { useMutation, useQueryClient } from '@tanstack/react-query';
+import type { ApiErrorResult } from '@dailydotdev/shared/src/graphql/common';
+import {
+  DEFAULT_ERROR,
+  gqlClient,
+} from '@dailydotdev/shared/src/graphql/common';
+import {
+  generateQueryKey,
+  RequestKey,
+} from '@dailydotdev/shared/src/lib/query';
+import { useAuthContext } from '@dailydotdev/shared/src/contexts/AuthContext';
+import { DELETE_ORGANIZATION_MUTATION } from '@dailydotdev/shared/src/features/organizations/graphql';
+import { settingsUrl } from '@dailydotdev/shared/src/lib/constants';
+import { useToastNotification } from '@dailydotdev/shared/src/hooks';
+import type { PromptOptions } from '@dailydotdev/shared/src/hooks/usePrompt';
+import { usePrompt } from '@dailydotdev/shared/src/hooks/usePrompt';
 import { getOrganizationLayout } from '../../../../components/layouts/OrganizationLayout';
+import { getTemplatedTitle } from '../../../../components/layouts/utils';
+import { defaultSeo } from '../../../../next-seo';
+import { AccountPageContainer } from '../../../../components/layouts/SettingsLayout/AccountPageContainer';
 
 const Page = (): ReactElement => {
   const router = useRouter();
-  const [imageChanged, setImageChanged] = useState(false);
+  const { user } = useAuthContext();
+  const { displayToast } = useToastNotification();
+  const { showPrompt } = usePrompt();
   const {
     organization,
     isFetching,
-    onUpdateOrganization,
+    updateOrganization,
     isUpdatingOrganization,
     seats,
+    isOwner,
   } = useOrganization(router.query.orgId as string);
+  const queryClient = useQueryClient();
+
+  const [imageChanged, setImageChanged] = useState(false);
+
+  const { mutateAsync: deleteOrganization, isPending: isDeletingOrganization } =
```

Comment:
Hmm maybe we should just add this to the hook that exports the rest already?

Reply:
Yeah, could do that

Reply:
Merged into the other hooks